nix/sources.nix

```diff
@@ -6,25 +6,33 @@ let
   # The fetchers. fetch_<type> fetches specs of type <type>.
   #
 
-  fetch_file = pkgs: spec:
-    if spec.builtin or true then
-      builtins_fetchurl { inherit (spec) url sha256; }
-    else
-      pkgs.fetchurl { inherit (spec) url sha256; };
+  fetch_file = pkgs: name: spec:
```

Comment:
Were these changes just pulled in from niv, or is there custom logic in here?

Reply:
I just run `niv init`. That’s just a version bump in the `niv` logic. Nothing manual happened in here.